http/src/routing/route.rs

```diff
@@ -1446,6 +1502,14 @@ impl<'a> Route<'a> {
             Self::GetGuildMembers { guild_id, .. } | Self::UpdateCurrentMember { guild_id, .. } => {
                 Path::GuildsIdMembers(*guild_id)
             }
+            Self::CreateGuildScheduledEvent { guild_id, .. }
+            | Self::DeleteGuildScheduledEvent { guild_id, .. }
+            | Self::GetGuildScheduledEvent { guild_id, .. }
+            | Self::GetGuildScheduledEventUsers { guild_id, .. }
+            | Self::GetGuildScheduledEvents { guild_id, .. }
+            | Self::UpdateGuildScheduledEvent { guild_id, .. } => {
+                Path::GuildsIdScheduledEvents(*guild_id)
+            }
```

Comment:
Should all of these map to the same `Path`? Cause in http-ratelimit there exists three types.